README.md

```diff
@@ -19,7 +19,16 @@
     ```sh
     npm install -g aicommits
     ```
+    If u can't install it as a global package, try installing it locally.
+    ```sh
+    npm install aicommits
+    ```
+    Then add the following to your `.bashrc` or `.zshrc` file:
+    ```sh
+    echo "alias aicommits=\"$(pwd)/node_modules/aicommits/dist/cli.mjs\"" >> ~/.bashrc
```

Comment:
If you're going to install it locally, you shouldn't point to it from a global configuration.

Also, you can use `npx aicommits` from the local project.